scripts/main.py

```diff
@@ -1,8 +1,37 @@
-import sys
-from image import parse_image
+from PIL import Image
+import numpy as np
 
-image = parse_image(sys.argv)
+def get_average_color(left, right, upper, lower):
+	r, g, b = 0, 0, 0
+	for x in range(left, right):
+		for y in range(upper, lower):
+			r += image.getpixel((x, y))[0]
+			g += image.getpixel((x, y))[1]
+			b += image.getpixel((x, y))[2]
+	pixel_count = chunk_width*chunk_height
+	r = int(r/pixel_count)
```

Comment:
Does `r /= pixel_count` work? Would be cleaner.

Reply:
I need to cast to int. This is fine, no?